### PR TITLE
feat: Images

### PR DESCRIPTION
**Pull request by gulam-mujtaba**
### Fifth Component Added to Component Library.

`Images`

**Features Added:**

1. Responsive Image - Square.
2. Responsive Image - Round.

**Screenshots**
image ss
image ss1

### CODE REVIEW

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
+@import url(/main.css);
+
+.main-container {
+  display: flex;
+}
+
+.sidebar {
+  height: auto;
+  background-color: black;
```
Comment (zabihshaik) [inline components/avatar.css:9]: Better to use variables for colors

File: components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  padding: none;
+  width: 25%;
+  display: fixed;
+}
+
+.sidebar h2 {
+  padding-left: 25px;
```
Comment (zabihshaik) [inline components/avatar.css:16]: Better to use rems instead of px

File: components/badges.css [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+}
+
+li a {
+  font-size: 1.4rem;
+  color: white;
+  height: 3rem;
+  text-align: center;
+  margin-top: 18px;
+  display: flex;
+  margin-left: 30px;
```
Comment (zabihshaik) [inline components/badges.css:29]: Better to use short-hand property for margin (margin: top right bottom left)

**Review by zabihshaik — COMMENTED** [review]
Everything looks. Just a few comments. Please check.

File: components/alert.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+.alert-secondary {
+  display: flex;
+  width: 600px;
+  padding: 0.5rem;
+  margin: 10px 0;
+  background-color: #ebe8e8;
+  border-radius: 5px;
+  font-weight: medium;
+}
+
```
Comment (NandhiniKrishnamoorthy) [inline components/alert.css:33]: alert-primary and alert-secondary has same style rules except background color, so you can write in a same class and reuse the code

File: components/alert.css [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+.alert-cancel {
+  display: flex;
+  width: 600px;
+  padding: 0.5rem;
+  margin: 10px 0;
+  background-color: #096427a1;
```
Comment (NandhiniKrishnamoorthy) [inline components/alert.css:75]: Please use variables for colors to keep consistency

File: components/images.html [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+            <h2 class="images-h2">DOV UI Images</h2>
+
+            <div class="image-container">
+                <h4>Square Shaped Image</h4>
+                <p class="image-des">Responsive Images can fit to the parent's width.
+                <br>Resize the bowser window to check the responsiveness of the image</p>
+                <img src="/resources/avatar.png" class="image-square">
+                <h4>Rounded Image</h4>
+                <p class="image-des">Images can also be customised to be round shaped</p>
+                <img class="image-round" src="/resources/avatar.png">
```
Comment (NandhiniKrishnamoorthy) [inline components/images.html:63]: Please use alt property for img

**Review by NandhiniKrishnamoorthy — APPROVED** [review]
Looks good. Use style guide to reuse the code

**Pull request merged**